packages/common/src/components/Inputs/Dropdown.tsx

```diff
@@ -14,17 +14,18 @@ interface Props {
 const Dropdown: React.FC<Props> = (props) => {
   const { onChange, label, name, id, value, hasValidationError, options, disabled } = props
   return (
-    <div
-      className={`ic-forms__select ${hasValidationError ? 'iu-border-error dropdown' : 'dropdown'} ${
-        disabled ? 'ic-forms__select--disabled' : ''
-      }`}>
-      {label !== 'undefined' ? <label>{label}</label> : null}
-      <select value={value} name={name} id={id} disabled={disabled} onChange={onChange}>
-        {options}
-      </select>
-    </div>
+    <>
+      {label !== 'undefined' ? <label htmlFor={id}>{label}</label> : null}
```

Comment:
If I remove this the text undefined will show in the LISJP and AG7804. I don't know why the value is sent has 'undefined' instead of undefined if a question doesn't have a label.